skyportal/tests/api/test_earthquake.py

```diff
@@ -0,0 +1,54 @@
+import os
+import numpy as np
+
+from skyportal.tests import api
+
+
+def test_earthquake(super_admin_token, view_only_token):
```

Comment:
Sounds like a button a super-villain would have on his dashboard.

Reply:
We are going to give you this button.